dynamodb-local/tables/0pdd-events.json

```diff
@@ -2,7 +2,7 @@
   "TableName": "0pdd-events",
   "AttributeDefinitions": [
     {
-      "AttributeName": "repo",
+      "AttributeName": "id",
```

Comment:
@mbao01 this is dangerous, we already have AWS Amazon scheman and won't change it. Why changing the name of the attribute?